client/state/themes/selectors/index.js

```diff
@@ -68,6 +69,9 @@ export { isWporgTheme } from 'calypso/state/themes/selectors/is-wporg-theme';
 export { prependThemeFilterKeys } from 'calypso/state/themes/selectors/prepend-theme-filter-keys';
 export { shouldShowTryAndCustomize } from 'calypso/state/themes/selectors/should-show-try-and-customize';
 export { shouldShowHomepageWarning } from 'calypso/state/themes/selectors/should-show-homepage-warning';
+export { shouldShowEligibilityWarning } from 'calypso/state/themes/selectors/should-show-eligibility-warning';
```

Comment:
Super-tiny nit: Can we move this one line up? This line should sort before the one above it. 🤓